client.go

```diff
@@ -140,6 +140,21 @@ func (c *Client) HLR(id string) (*HLR, error) {
 	return hlr, nil
 }
 
+// HLRs lists all HLR objects that were previously created by the NewHLR
+// function.
+func (c *Client) HLRs() (*HLRList, error) {
+	hlrList := &HLRList{}
+	if err := c.request(hlrList, "hlr", nil); err != nil {
```

Comment:
As `"hlr"` is not used used single time anymore you may want to introduce constant. Same goes for other string literals.